.pylintrc

```diff
@@ -62,7 +62,8 @@ disable=fixme,
         superfluous-parens,
         bad-continuation,
         line-too-long,
-        bad-option-value
+        bad-option-value,
+        assigning-non-slot
```

Comment:
why do we need to add `assigning-non-slot` to the disabled list?

Reply:
OIC this is due to `flask.g.*** =` and we're getting this error in all new CI runs now, eg [dependabot PRs](https://app.circleci.com/pipelines/github/plotly/dash/2245/workflows/412fba8d-f052-4de3-9fdc-23c14680608d/jobs/34927) and this is a result of changes in Flask v2.0 (👋 @anders-kiaer https://github.com/pallets/flask/issues/4020)

The good news is Dash works fine with Flask 2.0 🎉 

Rather than a global override of that rule, can we just skip it on the relevant lines with `# pylint: disable=assigning-non-slot` as in https://github.com/equinor/webviz-config/pull/451 ?

Reply:
> The good news is Dash works fine with Flask 2.0 🎉

Indeed 👍 - we were also glad to see Dash out of the box worked with Flask 2.0 🚀. 

Off topic: Slightly related to the `pylint` issue here - I _think_ Flask started using `__slots__` without `__dict__` on the `g` object for further performance improvements. I started wondering if using `__slots__` could be a useful and simple improvement idea for Dash as well (#1657) - not sure (I have no experience using `__slots__` directly myself as of today).